app/res/layout/question_widget_text.xml

```diff
@@ -4,5 +4,6 @@
     android:layout_height="match_parent"
     android:gravity="start"
     android:scrollHorizontally="false"
+    android:textAppearance="@style/TextAppearance.AppCompat.Body1"
```

Comment:
Can you confirm this doesn't cause any significant size difference as compared to earlier ?

Reply:
Just checked, I don't see any difference in the size.

Reply:
thanks for confirming.